docs/source/getting_started/sample_data.md

```diff
@@ -14,30 +14,44 @@ file_names = sample_data.list_datasets()
 print(file_names)
 ```
 
-This will print a list of file names containing sample pose data.
+This prints a list of file names containing sample pose data.
 Each file is prefixed with the name of the pose estimation software package
 that was used to generate it - either "DLC", "SLEAP", or "LP".
 
-To load one of the sample datasets, you can use the
+To load one of the sample datasets, use the
```

Comment:
```suggestion
To load one of the sample datasets as a
[movement dataset](target-dataset), use the
```
And then remove the first sentence after the code snippet.

Reply:
Nice!